orchestra/static/orchestra/task/js/autosave.js

```diff
@@ -0,0 +1,87 @@
+(function() {
```

Comment:
I feel like it's an autoSaver which wraps saving functionality—going to leave it as is, I think :blush: